bower.json

```diff
@@ -11,11 +11,11 @@
   "license": "Apache License, Version 2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/mtscout6/react-router-bootstrap.git"
+    "url": "git://github.com/react-router/react-router-bootstrap.git"
   },
   "dependencies": {
-    "react-router": "^0.11.6",
-    "react-bootstrap": "^0.13.2"
+    "react-bootstrap": ">=0.15",
+    "react-router": ">=0.13.1"
```

Comment:
You don't have classnames set up as an external in your webpack config: https://github.com/react-bootstrap/react-router-bootstrap/blob/master/webpack.config.js#L22

Currently classnames just gets built into the distributed bundle and is not required as a bower dependency. I could update that if you want, but thought it was intentional.